.github/workflows/cypress.yml

```diff
@@ -17,12 +23,15 @@ jobs:
       - name: Raise for error
         if: env.BASE_URL == ''
         run: exit 1
-    outputs:
-      BASE_URL: ${{ env.BASE_URL }}
-  Chrome:
-    runs-on: ubuntu-latest
-    needs: Setup
-    steps:
+      - name: Raise for maintenance mode
+        run: |
+          html_body=$(curl ${{ env.BASE_URL }})
+          if echo $html_body | grep -q "Offline for Maintenance";
+          then
+            exit 1
+          else
+            echo "NOT in maintenance. Likely Heroku trigger after successful deploy";
+          fi
```

Comment:
Test the URL to see if the app is not up yet (first commit) or if the app is in maintenance mode (waiting on deploy to finish)